configs/upstreams.conf

```diff
@@ -130,3 +130,23 @@ upstream technical-documentation-page {
 upstream tribute-page {
   server localhost:50270;
 }
+
+upstream 25--5-clock {
+  server localhost:50005;
+}
+
+upstream drum-machine {
+  server localhost:50030;
```

Comment:
```suggestion
  server localhost:50038;
```

Reply:
I was doing with the port file open on the side 😓😓